### DISCUSSION TITLE
关于8310/8268等ID卡过防火墙原理

### DISCUSSION THREAD

**Discussion by frank-pian** [wh201906/Proxmark3GUI#35] [General]
一直想知道8310/8268这些卡片的写入方式,tb上已经有很多便宜的写入器了，说明原理层面上应该很简单，但是proxmark一直没有支持。
所以想讨论以下。
1. ID防火墙检测原理
是否跟IC卡一样，检测方式就是收集写入指令，只要能成功修改卡数据，就证明是复制卡，因此T5577是否有办法写入后关闭写入指令，从而一样做到过防火墙
2. 8310/8268卡的写入方法
我拆开了一个8310卡扣，里面用的芯片上写的是F08，F08不是复旦F08那个国产M1卡芯片？因此他有什么办法可以改为低频卡芯片？

**Comment by wh201906** [comment]
关于问题1，T5577卡也是可以过防火墙的，你需要复制之后给T5577卡设置一个密码，可能还需要改一下downlink模式。
关于问题2，高频卡肯定不可能改成低频卡芯片的，所以芯片上面的F08应该和复旦F08没太大关系。

**Reply by frank-pian** [reply]
感谢大佬回复。
家里的楼下大门我用T5577设置了密码，但是没什么用，8268卡也不行，唯独8310卡可以过，这中间看来还有什么机制不清楚，downlink模式大概是什么意思呢？

**Reply by wh201906** [reply]
很久没玩有点忘了，反正也是影响读写T5577卡的参数之一。在用市面上很常见的小蓝枪把ID卡复制到T5577卡的时候它会给T5577卡设置密码，同时也会修改downlink模式。
具体修改方法你可以在命令行下面看一下帮助
